test/e2e/csv_e2e_test.go

```diff
@@ -3,52 +3,41 @@ package e2e
 import (
 	"context"
 	"fmt"
-	"strconv"
```

Comment:
This file has a really big diff so it's hard to tell what's going on but if the e2e performance has been improved then we should get it in. I will try it out locally and then lgtm